Src/FluentAssertions/Types/MethodInfoAssertions.cs

```diff
@@ -44,6 +44,31 @@ public MethodInfoAssertions(MethodInfo methodInfo)
         }
 
         /// <summary>
+        /// Asserts that the selected method is not virtual.
+        /// </summary>
+        /// <param name="because">
+        /// A formatted phrase as is supported by <see cref="string.Format(string,object[])" /> explaining why the assertion 
+        /// is needed. If the phrase does not start with the word <i>because</i>, it is prepended automatically.
+        /// </param>
+        /// <param name="becauseArgs">
+        /// Zero or more objects to format using the placeholders in <see cref="because" />.
+        /// </param>
+        public AndConstraint<MethodInfoAssertions> NotBeVirtual(
+            string because = "", params object[] becauseArgs)
+        {
+            string failureMessage = "Expected method " +
```

Comment:
Why not put this on a single line, preferably using string interpolation? Same for similar uses elsewhere in this PR.